studio/src/aem/content-navigation.js

```diff
@@ -1,6 +1,10 @@
 import { css, html, LitElement, nothing } from 'lit';
 import { styleMap } from 'lit/directives/style-map.js';
 import { EVENT_CHANGE, EVENT_LOAD } from '../events.js';
+import { deeplink, pushState } from '../deeplink.js';
+import { getTopFolder } from './aem-fragments.js';
```

Comment:
I think `getTopFolder` should be in a different file.
since `aem-fragments.js` doesn't belong to content-navigation, it should not be directly imported IMO.